src/platform/common/utils/localize.ts

```diff
@@ -1396,6 +1396,18 @@ export namespace DataScience {
         );
     export const listOfFilesWithLinksThatMightNeedToBeRenamed = () =>
         localize('DataScience.listOfFilesWithLinksThatMightNeedToBeRenamed', 'File(s): {0} might need to be renamed.');
+    export const noActiveKernelSession = () =>
+        localize('DataScience.noActiveKernelSession', 'No no active kernel session.');
+    export const failedToGetVersionOfPandas = () =>
+        localize(
+            { key: 'DataScience.failedToGetVersionOfPandas', comment: ['{Locked="Pandas"}'] },
+            'Failed to get version of Pandas to use the Data Viewer'
```

Comment:
I think this message needs to end with a period.

Reply:
Coudln't find changes to package.nls. in this PR for any of these. or is it in here and I'm missing it.